jenkins-scripts/dsl/tools/get_collections_from_package_and_version.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python3
+
+import sys
+import yaml
+
+
+# Function to find the collection name based on lib name and major version
+def find_collection(data, lib_name, major_version):
+  instances = []
+
+  for collection in data['collections']:
+    for lib in collection['libs']:
+      if lib['name'] == lib_name and lib['major_version'] == major_version:
+        instances.append(collection['name'])
+  return instances
+
+
+def get_major_version(version):
+  elements = version.split('.')
+  return int(elements[0])
+
+
+if len(sys.argv) < 3:
+  print(f"Usage: {sys.argv[0]} <lib_name> <major_version> <collection-yaml-file>")
```

Comment:
I won't block on this, but my preference would be to put more of the usage info in here rather than README.md. Users are accustomed to running a script or running with `-h` to see usage information.

Reply:
That is also a good point but given that this repo uses README descriptions extensively, I'm going to to keep it consistent and store the details in as they are in the file.